app/src/main/java/org/sopt/havit/ui/contents/ContentsActivity.kt

```diff
@@ -318,10 +316,31 @@ class ContentsActivity : BaseBindingActivity<ActivityContentsBinding>(R.layout.a
         toast.show()
     }
 
-    private fun setToast(){
-        contentsAdapter.setHavitClickListener(object : ContentsAdapter.OnHavitClickListener {
-            override fun onHavitClick() {
-                setCustomToast()
+    // 해빗 클릭 시 이벤트 함수 정의
+    private fun clickItemHavit(){
+        contentsAdapter.setHavitClickListener(object : ContentsAdapter.OnItemHavitClickListener {
+            override fun onHavitClick(v: ImageView, position: Int) {
+                with(contentsAdapter) {
+                    // 보지 않았던 콘텐츠의 경우 콘텐츠를 봤다는 토스트 띄우기
+                    if (!currentList[position].isSeen) {
+                        setCustomToast()
+                    }
+
+                    currentList[position].isSeen = !currentList[position].isSeen
+                    // 서버 호출
+                    contentsViewModel.setIsSeen(currentList[position].id)
+                    // 태그 바꾸기
+                    when(v.tag){
+                        "unseen" -> {
+                            v.tag = "seen"
+                            v.setImageResource(R.drawable.ic_contents_read_2)
+                        }
+                        else -> {
+                            v.tag = "unseen"
+                            v.setImageResource(R.drawable.ic_contents_unread)
+                        }
+                    }
```

Comment:
아래와 같은 로직으로 간결하게 나타낼 수도 있을 것같아요😊
```suggestion
var isSeen = (v.tag == "seen")
v.tag =if (isSeen) "unseen" else "seen"
v.setImageResource(if (isSeen) R.drawable.ic_contents_read_2 else R.drawable.ic_contents_unread)
```

Reply:
그냥 코드 확인 안해보고 코드리뷰 하면서 슬쩍 쓴 부분이라
Commit Suggestion 바로 적용하지 마시고,
직접 안드로이드 스튜디오에서 테스트하고 반영해주세요💜

Reply:
오오앙ㅇ 알겠습니닷!! 감사합니당~!